### PR TITLE
CORAAL asr1 recipe

### PR DESCRIPTION
**Pull request by kalvinchang**
## What?
* We introduce a new dataset of African American Vernacular English (AAVE) to ESPnet called CORAAL (Corpus of Regional African American Language) [1].
* We preprocess the data and train a Transformer + WavLM features model on the dataset.
* AAVE is a low resource variety on which ASR underperforms [2, 3].
    * Recent works [4-6] do not publicly release their preprocessed version of the dataset, making it hard to compare.

## Why?
* n/a - We did not modify anything pre-existing, except adding CORAAL to db.sh

## See also
* RESULTS.md for a description of CORAAL

## References
[1] T. Kendall and C. Farrington. 2023. The Corpus of Regional African American Language. Version 2023.06. Eugene, OR: The Online Resources for African American Language Project. [https://doi.org/10.7264/1ad5-6t35].

[2] A. Koenecke, A. Nam et al., "Racial disparities in automated speech recognition," *PNAS*, vol. 117, no. 14, pp. 7684–7689, 2020.

[3] K. Chang, Y.-H. Chou, J. Shi, H.-M. Chen, N. Holliday, O. Scharenborg, and D. Mortensen. "Self-supervised Speech Representations Still Struggle with African American Vernacular English," In *Proceedings of INTERSPEECH 2024*, 2024.

[4] Y. Zhang, W. Han, J. Qin, Y. Wang, A. Bapna, Z. Chen, N. Chen, B. Li, V. Axelrod, G. Wang, Z. Meng. Google USM: Scaling automatic speech recognition beyond 100 languages. arXiv preprint arXiv:2303.01037. 2023 Mar 2.

[5] A. Aksenova, Z. Chen et al., “Accented speech recognition: Benchmarking, pre-training, and diverse data,” *arXiv preprint* arXiv:2205.08014, 2022.

[6] S. Garg, Z. Huo et al., “Improving speech recognition for African American English with audio classification,” in *Proc. ICASSP*, 2024, pp. 12 356–12 360.

### CODE REVIEW

**Comment by sw005320** [comment]
Please fix the CI issue https://github.com/espnet/espnet/actions/runs/10601981829/job/29383660963?pr=5882

File: egs2/coraal/asr1/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# CORAAL (Corpus of Regional African American Language)
```
Comment (ftshijt) [inline egs2/coraal/asr1/README.md:1]: You can combine readme and results into a single file as the content are highly related.

File: egs2/coraal/asr1/run.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+nbpe=5000
+bpemode=unigram
+
+./asr.sh \
+    --lang en \
+    --stage 1 \
+    --ngpu 2 \
+    --nj 16 \
+    --gpu_inference true \
+    --inference_nj 2 \
```
Comment (ftshijt) [inline egs2/coraal/asr1/run.sh:27]: We can leave the settings to default.

**Review by ftshijt — COMMENTED** [review]
Thanks for the great contribution. Additionally, could you please also add coraal to egs2/README.md?

File: egs2/coraal/asr1/README.md (hunk 1 above)
Comment (kalvinchang) [reply egs2/coraal/asr1/README.md:1]: Sounds good!

File: egs2/coraal/asr1/run.sh (hunk 2 above)
Comment (kalvinchang) [reply egs2/coraal/asr1/run.sh]: sounds good!

Comment (kalvinchang) [reply egs2/coraal/asr1/run.sh]: should be fixed now :)

**Comment by kalvinchang** [comment]
@ftshijt thanks for reviewing this long PR :)

issues should be fixed.

**Comment by ftshijt** [comment]
Many thanks for your contribution! Will merge the PR.

**Pull request merged**